test/Project.toml

```diff
@@ -0,0 +1,14 @@
+[deps]
```

Comment:
It's not recommended to use test/Project.toml because its environments don't play nicely with compat handling of the main package.